src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDynamicTests.cs

```diff
@@ -683,8 +683,17 @@ public IEnumerable<dynamic> M1()
                             break;
 
                         case SymbolKind.Method:
-                            // Dev11 marks return type of GetEnumerator with DynamicAttribute, we don't
```

Comment:
Is that a breaking change? I don't understand how this is related to this PR.

Reply:
This is a result of moving the implementation of `AddSynthesizedReturnTypeAttributes` to `MethodSymbol`.

It's a change in behavior from the previous version of the compiler but these are synthesized methods implementing an interface on a synthesized class so any callers should be using the interface rather than the methods directly. From the comment, the native compiler generated attributes in these cases.